app/src/components/homePageMainContent/homePageMainContent.tsx

```diff
@@ -5,22 +5,10 @@ import SwimLanes from "../swimlanes/swimLanes";
 import SwimLanesLoading from "../swimlanes/swimLanesLoading";
 import { useNumColumns } from "../../hooks/useNumColumns";
 
-const HomePageMainContent = () => {
-  const numColumns = useNumColumns();
-  const [data, setData] = useState<any>({});
-
-  useEffect(() => {
-    const fetchData = async () => {
-      const response = await fetch("/api/homepage", {
-        method: "GET",
-        cache: "no-store",
-      });
-      const responseData = await response.json();
-      setData(responseData);
-    };
+const HomePageMainContent = (data) => {
```

Comment:
These are props and you need to destructure them, so this should be `({ data })`.

Reply:
The check on line 13 doesn't work because it's actually `data.data.lanesWithNumItems` in the way you have it set. So if you destructure the prop value, then the logic check will be correct.

Reply:
ah okay, that makes sense